tests/py_test.py

```diff
@@ -138,19 +137,19 @@ def op(self, db, ty):
     def run(self, db, op):
         [table] = db.run(db.sampler().all([('test', 'test_hist')]), op,
                          force=True, show_progress=False)
-        next(table.load([0], parsers.histograms))
+        next(table.load([1], parsers.histograms))
 
 @builder
 class TestOpticalFlow:
     def op(self, db, ty):
         input = db.ops.Input()
         flow = db.ops.OpticalFlow(
-            inputs=[(input,['frame', 'frame_info'])],
+            inputs=[(input,['index', 'frame', 'frame_info'])],
             device=ty)
         output = db.ops.Output(inputs=[(flow, ['flow']), (input, ['frame_info'])])
```

Comment:
This output should have the index column? Unless you are inserting it automatically.